redis_enterprise/tests/test_redis_enterprise.py

```diff
@@ -0,0 +1,29 @@
+from typing import Any, Callable, Dict  # noqa: F401
+
+import pytest
+
+from datadog_checks.dev.utils import get_metadata_metrics
+from datadog_checks.redis_enterprise import RedisEnterpriseCheck
+
+
+@pytest.mark.unit
+def test_check(dd_run_check, aggregator, instance):
+    # type: (Callable[[AgentCheck, bool], None], AggregatorStub, Dict[str, Any]) -> None
+    check = RedisEnterpriseCheck(
+        'redis_enterprise', {}, [{'openmetrics_endpoint': 'https://localhost:8070', 'tls_verify': 'false'}]
+    )
+    dd_run_check(check)
+
+    aggregator.assert_all_metrics_covered()
+    aggregator.assert_metrics_using_metadata(get_metadata_metrics())
```

Comment:
Interesting, does this pass for you locally?

Reply:
@j8-redis bumping this